dask/array/dispatch.py

```diff
@@ -11,3 +11,8 @@
 einsum_lookup = Dispatch("einsum")
 empty_lookup = Dispatch("empty")
 divide_lookup = Dispatch("divide")
+percentile_dispatch = Dispatch("percentile_dispatch")
```

Comment:
Just a minor point, but all the other `Dispatch` objects here have `*_lookup` instead of `*_dispatch`. For consistency could we update `percentile_dispatch` to `percentile_lookup`?